model/src/cfd.rs

```diff
@@ -378,6 +379,32 @@ impl CfdEvent {
             event,
         }
     }
+
+    /// Comparison function to order two events chronologically.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use model::{CfdEvent, Timestamp, EventKind, OrderId};
+    ///
+    /// # let id = OrderId::default();
+    /// # let make_event_at = move |seconds| CfdEvent {
+    /// #    timestamp: Timestamp::new(seconds),
+    /// #    id,
+    /// #    event: EventKind::LockConfirmed,
+    /// # };
+    /// #
```

Comment:
❓ Thanks for adding an example! What do the `#`s mean?

Reply:
They make this part of the code not render in rustdoc!